saiarcot895/sonic-buildimage
# DellEMC: Remove repeated files

#### Why I did it
ipmihelper files are repeated for few DellEMC platforms
#### How I did it
Removed the files in sonic_platform since as part of debian rules,ipmihelper will be copied to necessary directory.
#### How to verify it
Build the debian and verify whether the file is present in sonic_platform directory.
#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog


#### A picture of a cute animal (not mandatory but encouraged)
